application/web-ui/pom.xml

```diff
@@ -13,6 +13,28 @@
     </properties>
 
     <dependencies>
+        <!-- Spring Boot Starter Web -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <!-- Thymeleaf -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-thymeleaf</artifactId>
+        </dependency>
+        <!-- H2 Database (for in-memory database, optional) -->
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
Wordt niet gebruikt. 

Verwacht je ergens deze te gaan gebruiken? Voor cachen van pagina's misschien?

Reply:
Ook hier noemt die trouwens dat deze artifact niet bestaat.

Reply:
`1.769 [ERROR] 'dependencies.dependency.version' for com.h2database:h2:jar is missing. @ line 27, column 21`

Reply:
Thats odd, dat heb ik niet ervaren. Ik ga t uitzoeken

Reply:
Ja idd cachen als dat evt van toepassing is. Maar Ronald zei laatst al dat het geen fancy UI hoeft te zijn, gewoon een paar paginas om doorheen te klikken. Wordt niet beoordeeld op UI iig